Created the Dialogs pattern topic and added it to the TOC (#457)
IgniteUI/indigo-design-docfx: en/patterns/dialogs.md

---
title: Dialogs - Design System Pattern

> _description: The Dialogs Pattern Symbols are used to get user's attention and direct it toward important information and/or action they need to confirm and/or take.

Reviewer: ```suggestion
_description: The Dialog Patterns are used to direct the user's attention toward important information and/or action they need to take.
```

_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
---

# Dialogs

> Use the Dialogs Pattern to alert or give the user relevant information for an error or a system state that requires urgent user input. Dialogs are usually triggered by a user's action and should be displayed over the main content. It's best when dialogs are used sparingly as they are in a way distracting to the user.

Reviewer: ```suggestion
Use the Dialog Patterns to alert the users or give them relevant information for an error or a system state that requires urgent input or action. They are usually triggered by a user's action and should be displayed over the main content. Dialogs should be used sparingly as they interrupt the user in his experience with the product.
```

> <img class="responsive-img" src="../images/dialogs_demo.png" srcset="../images/dialogs_demo@2x.png 2x" />

Reviewer: Replace this image with an image of the first dialog because that is what you reference with the layers screenshot below.

Author: Done

> The Dialogs Pattern comes with the styling flexibility provided by the Avatar, Button, Checkbox, Input and List that constitute its layout.

Reviewer: ```suggestion
The Dialog Patterns come with the styling flexibility provided by the Buttons, Inputs, and List elements that constitute their layout.
```

> [!WARNING]

> > After inserting a Dialogs Pattern you should trigger `Detach from Symbol` in order to be able to generate it as Angular code. The individual Components, however, must stay intact and not be detached!

Reviewer: ```suggestion
> After inserting a Dialog Pattern, you should trigger `Detach from Symbol` in order to be able to generate it as Angular code. The individual Components, however, must stay intact and not be detached!
```

<img class="responsive-img" src="../images/dialogs_detach.png" srcset="../images/dialogs_detach@2x.png 2x" />

## Additional Resources

Related topics:

> - [Avatar](../components/avatar.md)
> - [Button](../components/button.md)
> - [Checkbox](../components/checkbox.md)

Reviewer: ```suggestion
- [Button](../components/button.md)
```
